docs/units-and-global-variables.rst

```diff
@@ -52,7 +52,7 @@ namespace and are mainly used to provide information about the blockchain.
 Block and Transaction Properties
 --------------------------------
 
-- ``block.blockhash(uint blockNumber) returns (bytes32)``: hash of the given block - only works for 256 most recent blocks excluding current
```

Comment:
Can't comment inline, but the `note` below still uses `block.blockhash`, should be updated, since the copy of that note in `miscellaneous` does the same.